src/hotspot/share/opto/vectorization.hpp

```diff
@@ -1315,4 +1318,11 @@ class AlignmentSolver {
 #endif
 };
 
+struct VTransformBoolTest {
+  const BoolTest::mask _mask;
+  const bool _is_negated;
+  VTransformBoolTest(const BoolTest::mask mask, bool is_negated) :
+    _mask(mask), _is_negated(is_negated) {}
+};
```

Comment:
Note: used by `PackSet::get_bool_test`. I gave it a `VTransform` name in anticipation of the `VTransform` refactoring in https://github.com/openjdk/jdk/pull/19261